### PR TITLE
cat follow mouse

### PR DESCRIPTION
**Pull request by crabby605**
my extension Cat-follow-mouse-real adds a cat on every webpage that can be pet and it follows ur mouse!!!

### CODE REVIEW

File: submissions/cat-folloe-mouse-real/catFollowMouse.js [hunk 1]
```diff
@@ -0,0 +1,281 @@
+(function oneko() {
```
Comment (JosiasAurel) [inline submissions/cat-folloe-mouse-real/catFollowMouse.js:1]: This is clearly a copy of [oneko.js](https://github.com/adryd325/oneko.js/blob/main/oneko.js). Copying other people's work is not acceptable.

**Review by JosiasAurel — CHANGES_REQUESTED** [review]
Please create something of your own or you need to make significant changes or add new features to what you currently have here.

**Comment by crabby605** [comment]
> Please create something of your own or you need to make significant changes or add new features to what you currently have here. 

I actually did mention that this uses oneko I nade it as a extension and added the petting function I'll add more if that's not enough

**Comment by JosiasAurel** [comment]
closing due to inactivity

**Pull request closed without merging**